oxidation/client/package.json

```diff
@@ -3,28 +3,22 @@
     "version": "0.0.1",
     "private": true,
     "scripts": {
-        "serve": "vue-cli-service serve",
         "postinstall": "cd electron && npm install",
         "test:unit": "vue-cli-service test:unit",
         "test:e2e": "vue-cli-service test:e2e",
         "lint": "vue-cli-service lint",
-        "build": "npx ionic build",
-        "android:copy": "npm run build && npx cap copy android",
-        "android:copy:release": "npm run build -- --prod && npx cap copy android",
+        "native:build": "vue-cli-service build",
+        "android:copy": "npm run native:build && npx cap copy android",
+        "android:copy:release": "npm run native:build -- --prod && npx cap copy android",
         "android:open": "npm run android:copy && npx cap open android",
-        "ios:copy": "npm run build && npx cap copy ios",
-        "ios:copy:release": "npm run build -- --prod && npx cap copy ios",
+        "ios:copy": "npm run native:build && npx cap copy ios",
+        "ios:copy:release": "npm run native:build -- --prod && npx cap copy ios",
         "ios:open": "npm run ios:copy && npx cap open ios",
-        "electron:copy": "npm run build && npx cap copy @capacitor-community/electron",
-        "electron:copy:release": "npm run build -- --prod && npx cap copy @capacitor-community/electron",
+        "electron:copy": "npm run native:build && npx cap copy @capacitor-community/electron",
+        "electron:copy:release": "npm run native:build -- --prod && npx cap copy @capacitor-community/electron",
         "electron:open": "npm run electron:copy && npx cap open @capacitor-community/electron",
         "electron:release": "npm run electron:copy:release && cd electron && npm run electron:release",
-        "web:libparsec:build": "cd ../bindings/web && wasm-pack build --dev --target web",
-        "web:libparsec:build:release": "cd ../bindings/web && wasm-pack build --release --target web",
-        "web:libparsec:copy": "cp -r ../bindings/web/pkg . || xcopy ..\\bindings\\web\\pkg pkg /e /s /y /i",
-        "web:copy": "npm run web:libparsec:build && npm run web:libparsec:copy",
-        "web:copy:release": "npm run web:libparsec:build:release && npm run web:libparsec:copy",
-        "web:open": "npm run web:copy && vue-cli-service serve",
+        "web:serve": "vue-cli-service serve",
```

Comment:
for consistency would be better to keep :open, like other platforms instead of serve

Reply:
as you like, done ;-)